### PR TITLE
movgrab: upstream Linux fix

### PR DESCRIPTION
**Pull request by danielnachun**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by iMichka** [comment]
I think we should bump the revision here as we are moving from brewed `libressl` to system `openssl` on Mac. Wondering why it was using `libressl` though.

**Review by iMichka — APPROVED** [review]

**Comment by SMillerDev** [comment]
any way we could improve the test here? I doubt the current one would fail with these (or any) changes

File: Formula/movgrab.rb [hunk 1]
```diff
@@ -14,7 +14,11 @@ class Movgrab < Formula
     sha256 cellar: :any, high_sierra:   "7702b7817fc398f4901014bd6162578294da414a18b1ae4e5f10ef8cf05a678c"
   end
 
-  depends_on "libressl"
```
Comment (carlocab) [inline Formula/movgrab.rb:17]: I think we want to keep this. This was added in https://github.com/Homebrew/homebrew-core/pull/49738, and the refereced issue (https://github.com/ColumPaget/Movgrab/issues/3) is still open.

Good chance that this just disables SSL support entirely. We can probably just drop the `revision` bump too.

Comment (danielnachun) [reply Formula/movgrab.rb:17]: I'm going to close this and align the Linux formula instead.

**Pull request closed without merging**